### PR TITLE
Lesson21-kubra

### PR DESCRIPTION
**Pull request by AysenurKubra**
I've changed just one line of codes that we did at class. We defined 'indicators' by using 'getElementsByClassName' ant it returns an HTMLcollection. However, we use 'querySelectorAll' for 'images' and it returns a nodeList. After a bunch of approaches, finally I realize this difference. I was trying to same function for both and getting error message from one of them. Then changed the code; it worked :)

### CODE REVIEW

File: Lesson21/Kubra/app.js [hunk 1]
```diff
@@ -3,49 +3,63 @@
...
-
+  nextButton.addEventListener('click', showNextImage);
+  previousButton.addEventListener('click', showPrevImage);
+  
 
   // Switch images after 3 seconds automatically using setInterval function
   // Use 3000 value for 3 second delay
-
+  
+  let timer = setInterval(showNextImage, 3000);
```
Comment (annasvst) [inline Lesson21/Kubra/app.js:64]: You don't need to assign `setInterval(showNextImage, 3000);` to a variable, you only need to call it once.
```suggestion
  setInterval(showNextImage, 3000);
```

Comment (AysenurKubra) [reply Lesson21/Kubra/app.js:64]: Actually, I know that it isn't necessary. The reason I assign to a var is that I was planning to add 'mouseover' and 'mouseout' to make it better. Kind of annoying when mouseover the picture it still continues auto skip. But it was late and I was tired at the end. I also think that maybe it won't be a good idea to add something extra which you didn't ask. Anyway:)) I am going to fix now. Thank you for your feedback!

Comment (annasvst) [reply Lesson21/Kubra/app.js]: I think it's actually a really good idea, feel free to add it and keep this as your own project, I'm pretty sure you can reuse this code in the future. :)

**Review by annasvst — APPROVED** [review]
Looks good, approved!

**Pull request merged**